src/main/java/controllers/dashBoardController.java

```diff
@@ -0,0 +1,262 @@
+        timeDialogPane.setContent(gridPane);
+        updateTime.setResultConverter((ButtonType button) -> {
+            if (button == ButtonType.OK && hourlist.getValue()!=null &&minlist.getValue()!=null) {
+                scheduleTimer.cancel();
+                scheduleTimer = new Timer(true);
+                LocalTime pickTime = LocalTime.of(hourlist.getValue(),minlist.getValue());
+                incrementTask =new IncrementTask();
+                incrementTask.setTime(pickTime);
+                scheduleTimer.scheduleAtFixedRate(incrementTask,1000,1000);
+            }
+            return null;
+        });
+        updateTime.showAndWait();
+    }
+
+    public void handleChangeDate(MouseEvent mouseEvent) {
+        Dialog<LocalDate> updateDate = new Dialog<>();
+        updateDate.setHeaderText("Change context date ");
+        DatePicker newDate= new DatePicker();
+        DialogPane dateDialogPane = updateDate.getDialogPane();
+        dateDialogPane.getButtonTypes().addAll(ButtonType.OK, ButtonType.CANCEL);
+        dateDialogPane.setContent(newDate);
+        updateDate.setResultConverter((ButtonType button) -> {
+            if (button == ButtonType.OK && newDate.getValue()!=null) {
+                // set the time
+                date.setText(newDate.getValue().toString());
+            }
+            return null;
+        });
+        updateDate.showAndWait();
+    }
+
+    public void handleChangeTemp(MouseEvent mouseEvent) {
+        Dialog<String> updateTemp = new Dialog<>();
+        updateTemp.setHeaderText("Change Outside Temperature");
+        JFXTextField newTemp = new JFXTextField();
+        JFXComboBox<String> sign =  new JFXComboBox();
+        ObservableList<String> symbol = FXCollections.observableArrayList();
+        sign.getItems().addAll("+","-");
+        DialogPane TempDialogPane = updateTemp.getDialogPane();
+        TempDialogPane.getButtonTypes().addAll(ButtonType.OK, ButtonType.CANCEL);
+        HBox content = new HBox();
+        content.getChildren().setAll(sign, newTemp);
+        TempDialogPane.setContent(content);
+        updateTemp.setResultConverter((ButtonType button) -> {
+
+            if (button == ButtonType.OK && newTemp.getText()!=null ) {
+                if(newTemp.getText().matches("[0-9]+")){
+                    if(sign.getValue()==null || sign.getValue().equals("+")){
+                        outsideT.setText(newTemp.getText());
+                    }
+                    else{
+                        outsideT.setText("-"+newTemp.getText());
+                    }
+                }
+                else{
+                    // cannot set the temperature keep old value and display on the log
+                }
+
+            }
+            return null;
+        });
+        updateTemp.showAndWait();
+    }
+
+    public void handleEdit(ActionEvent event) {
+
+    }
+    public void changeLoggedUser(MouseEvent mouseEvent) {
```

Comment:
Remove this method if not used. There is no code inside of it?

Reply:
> Remove this method if not used. There is no code inside of it?

changeLoggedUser will be implement when the second screen is finish. we will use it after
For edit, it would be used for edit screen